src/read.c

```diff
@@ -5589,6 +5589,12 @@ avifResult avifDecoderRead(avifDecoder * decoder, avifImage * image)
     if (result != AVIF_RESULT_OK) {
         return result;
     }
+    // If decoder->image->imageOwnsYUVPlanes and the decoder instance is not used after this call,
```

Comment:
Nit: Add "is true" after "decoder->image->imageOwnsYUVPlanes".

"the decoder instance is not used" can be more specific: "decoder->image is not used".